extensions/amp-consent/0.1/consent-config.js

```diff
@@ -320,15 +322,21 @@ export class ConsentConfig {
  * Expand consent endpoint url
  * @param {!Element|!ShadowRoot} element
  * @param {string} url
+ * @param {Object<string, *>=} vars
  * @return {!Promise<string>}
  */
-export function expandConsentEndpointUrl(element, url) {
+export function expandConsentEndpointUrl(element, url, vars = {}) {
   return Services.urlReplacementsForDoc(element).expandUrlAsync(
     url,
     {
       'CLIENT_ID': getConsentCID(element),
+      'CONSENT_PAGE_VIEW_ID_64': () =>
+        getServicePromiseForDoc(element, 'consentStateManager').then(
+          (consentStateManager) => consentStateManager.consentPageViewId64()
+        ),
+      ...vars,
     },
-    CONSENT_VARS_ALLOWED_LIST
+    {...vars, ...CONSENT_VARS_ALLOWED_LIST}
```

Comment:
Let's take advantage of the `...vars` allowlist spread for the variables we set here. We could save a few bytes and reduce the spilling of the value keys.

Change this implementation, and remove `CLIENT_ID` and `CONSENT_PAGE_VIEW_ID_64` from `CONSENT_VARS_ALLOWED_LIST`.

```js
  const vars = {
    'CLIENT_ID': getConsentCID(element),
    'CONSENT_PAGE_VIEW_ID_64': () =>
      getConsentStateManager(element).then(
        (consentStateManager) => consentStateManager.consentPageViewId64()
      ),
    ...opt_vars,
  };
  return Services.urlReplacementsForDoc(element).expandUrlAsync(
    url,
    vars,
    {...vars, ...CONSENT_VARS_ALLOWED_LIST}
  );
```